Data/Graph/Inductive/PatriciaTree.hs

```diff
@@ -30,6 +30,7 @@ import Data.Graph.Inductive.Graph
 import           Control.Applicative (liftA2)
 import           Data.IntMap         (IntMap)
 import qualified Data.IntMap         as IM
+import qualified Data.IntMap.Strict  as IMS
```

Comment:
This seems to fail with the version of containers shipped with GHC 7.4.2 (yet strangely works on earlier GHCs).